src/events/logic.py

```diff
@@ -227,6 +227,10 @@ class Events:
     # raised when a workflow element completes to hand over to the next one
     ON_WORKFLOW_ELEMENT_COMPLETE = 'on_workflow_element_complete'
 
+    # kwargs: request, article, article_access
+    # raised when an article is viewed or downloaded
```

Comment:
I'd add a comment mentioning that this is raised only for accesses that pass counter compliance checks.